hbase-common/src/main/java/org/apache/hadoop/hbase/CellComparatorImpl.java

```diff
@@ -448,6 +451,33 @@ public final int compareFamilies(Cell left, Cell right) {
       right.getFamilyArray(), right.getFamilyOffset(), right.getFamilyLength());
   }
 
+  /**
+   * This method will be overridden when we compare cells inner store to bypass family comparing.
+   */
+  protected int compareFamilies(KeyValue left, KeyValue right, int leftFamilyLength,
```

Comment:
Can you use more similar signature to existing methods? I find this ordering confusing, because everyone is used to doing left, leftOffset, leftLength, right, rightOffset, rightLenght. 

This goes for all the new methods.

Reply:
> Can you use more similar signature to existing methods? I find this ordering confusing, because everyone is used to doing left, leftOffset, leftLength, right, rightOffset, rightLenght.
> 
> This goes for all the new methods.

Make sense, will fix it later.
Thanks.